Makefile

```diff
@@ -0,0 +1,2 @@
+format: 
```

Comment:
**thought**: `make` is readily available in most OSes. Windows will require WSL2 to be installed.

Reply:
will add this info in readme file